app/scripts/lib/app-start.js

```diff
@@ -70,7 +71,7 @@ function Start(options = {}) {
   this._storage = options.storage || Storage;
   this._translator = options.translator;
   this._user = options.user;
-  this._window = options.window || window;
+  this._window = this.window = options.window || window;
```

Comment:
UserAgentMixin is used to read the `forceUA` query param for functional tests, it requires `this.window` rather than `this._window`. To avoid blowing the PR up too much I didn't update `this._window` to `this.window`, that can be a follow on.